### PR TITLE
added profile

### PR DESCRIPTION
**Pull request by ayushlohmod**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1423"></a>

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thanks for submitting a PR. Please revert the changes in the `package-lock.json` file.
Let us know if you need any help with that! Thanks. 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by ayushlohmod — COMMENTED** [review]
reverted the changes please review it!

**Comment by Panquesito7** [comment]
> reverted the changes please review it!

Hmm, I can still see the same `package-lock.json` changes in your PR. 👀

File: public/data/ayushlohmod.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://linkedin.com/in/ayushlohmod",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Full Stack development ",
```
Comment (Panquesito7) [inline public/data/ayushlohmod.json:25]: There's an extra space here. Might want to remove it.
```suggestion
      "title": "Started Full Stack development",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Comment by ayushlohmod** [comment]
Hey! @Panquesito7  can you tell me why this package-lock.json keep modified by itself

**Comment by emmalearnscode** [comment]
> Hey! @Panquesito7 can you tell me why this package-lock.json keep modified by itself

Sometimes it will automatically update the release when you do npm install. The important thing is to be careful when committing your code not to use the command `git add .` as then you accidentally commit the changes made to the file. Only ever add and commit the files you want to by using more granular commands like 'git add public' to only add changes made within the public folder (and as outlined in the instructions on the contributing page).

You should be able to fix the changes to package-lock by copying the file from the Eddiehub repo and pasting it into your fork. When you do that and commit the changes the files should match and that should remove it from the pull requesst.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by ayushlohmod** [comment]
Thanks @

> > Hey! @Panquesito7 can you tell me why this package-lock.json keep modified by itself
> 
> Sometimes it will automatically update the release when you do npm install. The important thing is to be careful when committing your code not to use the command `git add .` as then you accidentally commit the changes made to the file. Only ever add and commit the files you want to by using more granular commands like 'git add public' to only add changes made within the public folder (and as outlined in the instructions on the contributing page).
> 
> You should be able to fix the changes to package-lock by copying the file from the Eddiehub repo and pasting it into your fork. When you do that and commit the changes the files should match and that should remove it from the pull requesst.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

**Comment by ayushlohmod** [comment]
Thanks @emmalearnscode for the help i do git add . all this time

**Pull request closed without merging**